www/docs/en/dev/config_ref/index.md

```diff
@@ -331,12 +331,15 @@ AndroidGradlePluginGoogleServicesVersion<br/>{% cdv_vartype semver %} {% cdv_pla
 AndroidGradlePluginGoogleServicesEnabled<br/>{% cdv_vartype boolean %} {% cdv_platform android %} | *Default: false* <br/>Enables Google Services. The `google-services.json` file will be required.
 GradlePluginKotlinEnabled<br/>{% cdv_vartype boolean %} {% cdv_platform android %} | *Default: false* <br/>Enables Kotlin plugin for Kotline support.
 AppendUserAgent<br/>{% cdv_vartype string %} {% cdv_platform android %} {% cdv_platform ios %} | If set, the value will append to the end of old UserAgent of webview. When using with OverrideUserAgent, this value will be ignored.
-BackgroundColor<br/>{% cdv_vartype string %} {% cdv_platform android %} {% cdv_platform ios %} | Sets the app's background color. Supports a four-byte hex value, with the first byte representing the alpha channel, and standard RGB values for the following three bytes. <br/>__Note__: `transparent` value will set the application tile background to the accent color on Windows.
+AutoHideSplashScreen<br/> {% cdv_vartype boolean %} {% cdv_platform android %} {% cdv_platform ios %} | *Default: true* <br/> Indicates whether to hide splash screen automatically or not. The splash screen is hidden after the amount of time specified in the `SplashScreenDelay` preference.
+BackgroundColor<br/>{% cdv_vartype string %} {% cdv_platform android %} {% cdv_platform ios %} | Sets the app's background color. Supports a four-byte hex value, with the first byte representing the alpha channel, and standard RGB values for the following three bytes.
```

Comment:
I think `BackgroundColor` preference is no longer valid for Android.

Android uses preference `AndroidWindowSplashScreenIconBackgroundColor`.

`BackgroundColor` is also missing from this page: https://github.com/apache/cordova-docs/blob/master/www/docs/en/dev/core/features/splashscreen/index.md

Reply:
This is the preference for the app background colour, not the splashscreen background colour... apparently it was missing this whole time